Pinta.Core/Actions/ViewActions.cs

```diff
@@ -140,11 +140,13 @@ public void CreateMainMenu (Gtk.Menu menu)
 			ImageMenuItem zoomin = ZoomIn.CreateAcceleratedMenuItem (Gdk.Key.plus, Gdk.ModifierType.ControlMask);
 			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.equal, Gdk.ModifierType.ControlMask, AccelFlags.Visible));
 			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.KP_Add, Gdk.ModifierType.ControlMask, AccelFlags.Visible));
+			zoomin.AddAccelerator ("activate", PintaCore.Actions.AccelGroup, new AccelKey (Gdk.Key.KP_Add, 0, AccelFlags.Visible));
```

Comment:
Should there also be a matching accelerator for the `equal` key? (and `underscore` for the zoom out case). There are shortcuts with Ctrl for both.

Reply:
I tried to do the same with underscore key, but it doesn't work well without CTRL. Equal key works fine.

Reply:
Fixed it